soundata/datasets/starss2022.py

```diff
@@ -3,7 +3,7 @@
 .. admonition:: Dataset Info
     :class: dropdown
 
-    *Sony-TAu Realistic Spatial Soundscapes: sound scenes in various rooms and environments, together with temporal and spatial annotations of prominent events belonging to a set of target classes.
+    **Sony-TAu Realistic Spatial Soundscapes:** sound scenes in various rooms and environments, together with temporal and spatial annotations of prominent events belonging to a set of target classes.
 
     *Created By:*
         | Archontis Politis, Parthasaarathy Sudarsanam, Sharath Adavanne, Daniel Krause, Tuomas Virtanen
```

Comment:
Remove `|` as in [dcase23_task4b.py](https://github.com/soundata/soundata/pull/165/files#diff-ea6737eb033c8c11fd6da408f6e73877aad3dbad484c1079a466d4dc6b398707)

Reply:
you mean those in lines 9-10? Because in dcase23_task4b we also have:
```
*Created By:*
    | Annamaria Mesaros, Tuomas Heittola, and Tuomas Virtanen.
    | Tampere University of Technology. 
```

Reply:
yes, the comment is at line 9, in fact (github is focusing at the diff part for the PR)

Reply:
so all of these `|` should be removed? then also in dcase23_task4b?

Reply:
yes, I would remove all `|` since they are not being rendered in the documentation